### PR TITLE
Continuos integration with package -t:5000

### PR DESCRIPTION
**Pull request by Maosorio92**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,5 +31,6 @@
   },
   "dependencies": {
     "axios": "^0.27.2"
-  }
+  },
+  "-t":"5000"
```
Comment (manuelq12) [inline package.json]: This is wrong @Maosorio92 , you should edit the script "test" and add the -t 5000

Comment (manuelq12) [reply package.json]: To be more accurate, in line 7 of this file

**Review by manuelq12 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Maosorio92) [reply package.json]: Ok, thank you!!!, but, which one of the following must I apply?:
1. "test": "mocha -t 5000" or
2. "test": "-t 5000"

Thanks in advance!

Comment (manuelq12) [reply package.json]: The first one, is an additional parameter to the execution!

**Review by manuelq12 — APPROVED** [review]

**Pull request merged**